rust/lance-index/src/scalar/inverted/index.rs

```diff
@@ -2084,30 +2094,30 @@ pub struct DocSet {
 impl DocSet {
     #[inline]
     pub fn len(&self) -> usize {
-        self.row_ids.len()
+        self.row_addrs.len()
     }
 
     pub fn is_empty(&self) -> bool {
         self.len() == 0
     }
 
     pub fn iter(&self) -> impl Iterator<Item = (&u64, &u32)> {
-        self.row_ids.iter().zip(self.num_tokens.iter())
+        self.row_addrs.iter().zip(self.num_tokens.iter())
     }
 
-    pub fn row_id(&self, doc_id: u32) -> u64 {
-        self.row_ids[doc_id as usize]
+    pub fn row_addr(&self, doc_id: u32) -> u64 {
+        self.row_addrs[doc_id as usize]
     }
 
-    pub fn doc_id(&self, row_id: u64) -> Option<u64> {
+    pub fn doc_id(&self, row_addr: u64) -> Option<u64> {
```

Comment:
IMO, the `doc_id` is a good name; it's not necessary to be renamed to `doc_addr`.